### ISSUE TITLE
Unable to complete iDeal payment through steam on Linux

### ISSUE THREAD

**Issue by sankasan** [ValveSoftware/steam-for-linux#4684]
#### Your system information

* Steam client version: *The latest (but the issue is active for over an year!)*
* Distribution (e.g. Ubuntu): *Arch*
* Opted into Steam client beta?: [Yes/No] *Yes*
* Have you checked for system updates?: [Yes/No] *Yes*

#### Please describe your issue in as much detail as possible:
Unable to complete a purchase via the steam client. This is a long standing issue which I've tried to tackle with the steam support team. They redirected me here. For more information please look at the link below for a detailed history.

https://support.steampowered.com/view.php?ticketref=3382-EPSK-4853

#### Steps for reproducing this issue:

1. Launch steam client
2. Buy a game
3. Select iDeal
4. Select Triodos Bank
5. Wait a while
6. See the following error code:
> Error Code: -101
>Unable to connect to server. Server may be offline or you may not be connected to the internet.

**Comment by Plagman** [comment]
Are you able to check if this works on Windows for you?

**Comment by sankasan** [comment]
I just checked and experienced the same issue in Windows.

**Comment by Plagman** [comment]
Does using a full browser work on both platforms?

**Comment by sankasan** [comment]
Yes that does work on both platforms (took me a while to test on Windows).

**Comment by sankasan** [comment]
Issue still present in current version of Steam. Any update on this?

**Comment by ardje** [comment]
To be clear, I noticed in big picture mode the browser was not getting the payment session data, which is part of the URI.
I recall I was able to pay with the desktop client, but not with the big picture client.
I can try desktop again, but desktop mode is a few reboots away... :-(.
To work around it I just put money on the account using a normal browser.
Ohhh wait... I have a totally different failure... I get a 404 due to the third party payment processor not getting the session data... Not an unable to connect to server...
Ok, sorry my bad: I have a different bank (rabobank) that probably has different requirements, maybe the triodos bank uses websockets.
On ideal payment the actual payment is done after a redirect to the users own ideal bank portal.
I will file a bug for the big picture session data issue, but I thought I saw something related that it might have been fixed.
This bug however is after the initial ideal portal, going to the portal of the bank itself. I will try a payment using my bank.

**Comment by ardje** [comment]
Ok, I've tried steam client ideal payment using the rabobank, and the rabobank backend works.
The difference with TS is that I select ideal, then I select rabobank, then I select proceed. Then a popup appears.
In his case it seems that just selecting triodos bank is all it needs. That should be sniffable.

**Comment by rhymoid** [comment]
As a fellow Triodos customer, I can confirm this issue under Debian buster.
